Makefile

```diff
@@ -9,7 +9,7 @@ build:
 ifndef GORELEASER
 	$(error "goreleaser not found (`go get -u -v github.com/goreleaser/goreleaser` to fix)")
 endif
-	$(GORELEASER) --skip-publish --rm-dist --snapshot
+	$(GORELEASER) --rm-dist --snapshot
```

Comment:
Hi, although I'm not a maintainer of this project, I appreciate your effort here!

Probably this is a nit, but for anyone like me who wants to run `make build` on your own machine to just test a binary and a docker image locally, I wish there's a way to turn off automated-publishing.